docs/src/explanations/connectable.md

```diff
@@ -0,0 +1,666 @@
+
+This means that users can try to connect to input ports of their module! If I write `x :<>= y`, and `x` is an input to the current module, then that is what the connection is trying to do. However, because input ports are not connectable from within the current module, Chisel will throw an error. This is the same error a user would get using a mono-directioned operator: `x := y` will throw the same error if `x` is an input to the current module. *Whether a component is connectable is irrelevant to the semantics of any connection operator connecting to it.*
+
+In summary, the port-direction computation is relative to the root marked `IO`, but connection-direction computation is relative to the consumer/producer that the connection is doing. This has the positive property that connection semantics are solely based on the Chisel structural type and its relative alignments of the consumer/producer (nothing more, nothing less).
+
+### Aligned connection operator (:<=)
+
+For connections where you want the aligned-half of 'bulk-connect-like-semantics' where the aligned members are driven producer-to-consumer and flipped members are ignored, use `:<=` (the "aligned connection").
+
+```scala mdoc:silent
+class Example2 extends RawModule {
+  val incoming = IO(Flipped(new MixedAlignmentBundle))
+  val outgoing = IO(new MixedAlignmentBundle)
+  incoming.flippedChild := DontCare // Otherwise FIRRTL throws an uninitialization error
+  outgoing :<= incoming
+}
+```
+
+This generates the following Verilog, where the aligned members are driven `incoming` to `outgoing` and flipped members are ignored:
+
+```scala mdoc:verilog
+import chisel3.stage.ChiselStage
+
+ChiselStage.emitVerilog(new Example2())
+```
+
+### Flipped connection operator (:>=)
+
+For connections where you want the flipped-half of 'bulk-connect-like-semantics' where the aligned members are ignored and flipped members are connected consumer-to-producer, use `:<=` (the "flipped connection", or "backpressure connection").
+
+```scala mdoc:silent
+class Example3 extends RawModule {
+  val incoming = IO(Flipped(new MixedAlignmentBundle))
+  val outgoing = IO(new MixedAlignmentBundle)
+  outgoing.alignedChild := DontCare // Otherwise FIRRTL throws an uninitialization error
+  outgoing :>= incoming
+}
+```
+
+This generates the following Verilog, where the aligned members are ignore and the flipped members are driven `outgoing` to `incoming`:
+
+```scala mdoc:verilog
+import chisel3.stage.ChiselStage
+
+ChiselStage.emitVerilog(new Example3())
```

Comment:
```suggestion
getVerilogString(new Example3)
```

Reply:
I think you should do this for all of them but I ran out of steam suggesting the change.